dxr/testing.py

```diff
@@ -267,11 +265,12 @@ def config_input(cls, config_dir_path):
 
     @classmethod
     def teardown_class(cls):
-        if cls.should_delete_instance:
+        if cls.stop_for_interaction:
+            print "Pausing for interaction at 0.0.0.0:8000..."
```

Comment:
Funnily, I thought that Flask would tell us when it launches the server, but it didn't in this case.